lms/envs/aws.py

```diff
@@ -824,6 +824,10 @@
     ECOMMERCE_SERVICE_WORKER_USERNAME
 )
 
+##### JOURNALS CONFIGURATION SETTINGS #####
+JOURNALS_URL_ROOT = ENV_TOKENS.get('JOURNALS_URL_ROOT', JOURNALS_URL_ROOT)
```

Comment:
**Use Django App Plugin:** DAP will allow you to add your own Django Settings without needing to update the platform.

Reply:
fixed